charts/asset-manager/templates/nginx-configmap.yaml

```diff
@@ -119,6 +121,10 @@ data:
         error_log /dev/stderr;
 
         location / {
+          proxy_busy_buffers_size 16k;
```

Comment:
You'll probably want to align this with https://github.com/alphagov/govuk-helm-charts/blob/1230472/charts/generic-govuk-app/templates/nginx-configmap.yaml#L31 otherwise nginx will be needlessly spooling a lot of requests to and from disk (yup, nginx is full of lovely surprises like that 😅)

Reply:
(incidentally there should be no need to set `proxy_busy_buffers_size` if you're setting the other two options from that link)